src/System.Memory/src/System/Buffers/Text/Utf8Parser/Utf8Parser.Boolean.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Buffers.Text
+{
+    public static partial class Utf8Parser
+    {
+        /// <summary>
+        /// Parses a Boolean at the start of a Utf8 string.
+        /// </summary>
+        /// <param name="text">The Utf8 string to parse</param>
+        /// <param name="value">Receives the parsed value</param>
+        /// <param name="bytesConsumed">On a successful parse, receives the length in bytes of the substring that was parsed </param>
+        /// <param name="standardFormat">Expected format of the Utf8 string</param>
+        /// <returns>
+        /// true for success. "bytesConsumed" contains the length in bytes of the substring that was parsed.
+        /// false if the string was not syntactically valid or an overflow or underflow occurred. "bytesConsumed" is set to 0. 
+        /// </returns>
+        /// <remarks>
+        /// Formats supported:
+        ///     G (default)   True/False
+        ///     l             true/false
+        /// </remarks>
+        /// <exceptions>
+        /// <cref>System.FormatException</cref> if the format is not valid for this data type.
+        /// </exceptions>
+        public static bool TryParse(ReadOnlySpan<byte> text, out bool value, out int bytesConsumed, char standardFormat = default)
+        {
+            if (!(standardFormat == default(char) || standardFormat == 'G' || standardFormat == 'l'))
+                throw new FormatException(SR.Argument_BadFormatSpecifier);
+
+            if (text.Length >= 4)
```

Comment:
This implementation doesn't produce different results based on the standardFormat.
Is any upper/lower case combination of the characters within `true` considered valid for both 'G' and 'l'?

Reply:
Yes.